lib/rhc/commands/app.rb

```diff
@@ -376,9 +376,16 @@ def tidy(app)
     syntax "<app> [--namespace NAME]"
     takes_application :argument => true
     def enable_ha(app)
-      app_action :enable_ha
+      rest_app = find_app
+      result = rest_app.send :enable_ha
 
-      results { say "#{app} is now highly available" }
+      results do
+        say "#{app} is now highly available"
+        # If the application was already scaled, a scale up may be required.
+        if rest_app.attributes["gear_count"] > 1
+          say "You may need to scale up the application to create an additional haproxy instance."
+        end
```

Comment:
This logic is incorrect. What we need to check is the number of gears in the web_framework group instance. The total application gears could include database gears (separate group instance) and should not be considered here.